### PR TITLE
Release 0.1.0a1

### PR DESCRIPTION
**Pull request by fao89**
I did some test and I noticed it  would break the sphinx on the first release

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -22,7 +22,3 @@ Features
 - Download a package with its dependencies
   `#6004 <https://pulp.plan.io/issues/6004>`_
 
-
-----
```
Comment (fao89) [inline CHANGES.rst]: @daviddavis @dralley @mdellweg towncrier adds `----` at the end which breaks sphinx, the first release needs some sphinx tweaks

**Review by dralley — APPROVED** [review]

**Comment by fao89** [comment]
merg after:
- https://github.com/pulp/pulp_npm/pull/27
- https://github.com/pulp/pulp_npm/pull/28

File: .bumpversion.cfg [hunk 2]
```diff
@@ -1,16 +1,16 @@
 [bumpversion]
-current_version = 0.1.0a1.dev
+current_version = 0.1.0a2.dev
 commit = False
 tag = False
 parse = (?P<major>\d+)\.(?P<minor>\d+)\.(0a)?(?P<patch>\d+)(\.(?P<release>[a-z]+))?
-serialize =
+serialize =
```
Comment (mdellweg) [inline .bumpversion.cfg:6]:
```suggestion
serialize =
```

File: .bumpversion.cfg [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
-serialize =
+serialize = 
 	{major}.{minor}.0a{patch}.{release}
 	{major}.{minor}.0a{patch}
 
 [bumpversion:part:release]
 optional_value = prod
 first_value = dev
-values =
+values =
```
Comment (mdellweg) [inline .bumpversion.cfg:13]:
```suggestion
values =
```

File: .bumpversion.cfg (hunk 2 above)
Comment (fao89) [reply .bumpversion.cfg:6]: it was done by bump2version, I just ran: 
```
python .github/release.py patch --lower 3.7 --upper 3.10
```
and rebased to fix the docs

Comment (mdellweg) [reply .bumpversion.cfg:6]: Interesting. That tool was not even supposed to touch that line, right?

Comment (fao89) [reply .bumpversion.cfg:6]: I believe so

**Pull request merged**